plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/RoleComponent.groovy

```diff
@@ -170,7 +171,8 @@ extends DirectComponentSupport
     securitySystem.getAuthorizationManager(DEFAULT_SOURCE).deleteRole(id)
   }
 
-  private static RoleXO asRoleXO(Role input) {
+  @PackageScope
```

Comment:
applying this and removing private modifier works, and is the lowest barrier to fixing the problem while maintaining some privacy